cra/sample/pr-pipeline.yaml

```diff
@@ -0,0 +1,326 @@
+        - name: secrets
+          workspace: artifacts
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: repository-integration
+          value: $(params.repository-integration)
+        - name: ibmcloud-api-key-secret-key
+          value: api-key
+        - name: pipeline-debug
+          value: $(params.pipeline-debug)
+
+    - name: code-fetch-code
+      runAfter:
+        - code-fetch-credentials
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: artifacts
+          workspace: artifacts
+        - name: secrets
+          workspace: artifacts
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: branch
+          value: $(params.branch)
+        - name: git-user
+          value: $(tasks.code-fetch-credentials.results.git-auth-user)
+        - name: pipeline-debug
+          value: $(params.pipeline-debug)
+
+    - name: cra-discovery-scan
+      runAfter:
+        - code-fetch-code
+      taskRef:
+        name: cra-discovery
+      workspaces:
+        - name: artifacts
+          workspace: artifacts
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: revision
+          value: $(params.branch)
+        - name: commit-id
+          value: $(params.commit-id)
+        - name: commit-timestamp
+          value: $(params.commit-timestamp)
+        - name: ibmcloud-api-key
+          value: $(params.api-key)
+        - name: pipeline-debug
+          value: $(params.pipeline-debug)
+        - name: directory-name
+          value: $(tasks.code-fetch-code.results.directory-name)
+
+    - name: code-vulnerability-scan-status-pending
+      runAfter:
+        - code-fetch-code
+      workspaces:
+        - name: artifacts
+          workspace: artifacts
+      taskRef:
+        name: git-set-commit-status
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: revision
+          value: $(params.commit-id)
+        - name: state
+          value: pending
+        - name: context
+          value: cra-vulnerability-scan
+        - name: description
+          value: "Tekton PR Pipeline Status"
+        - name: pipeline-debug
+          value: $(params.pipeline-debug)
+
+    - name: cra-vulnerability-scan
+      runAfter:
+        - cra-discovery-scan
+      taskRef:
+        name: cra-vulnerability-remediation
+      workspaces:
+        - name: artifacts
+          workspace: artifacts
+        - name: secrets
+          workspace: artifacts
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: revision
+          value: $(params.branch)
+        - name: pr-url
+          value: $(params.pr-url)
+        - name: commit-id
+          value: $(params.commit-id)
+        - name: git-api-token-key
+          value: git-token
+        - name: ibmcloud-api-key
+          value: $(params.api-key)
```

Comment:
Hum... are we dealing here with the real value for api key to use ? not the indirection in the secret secure-properties by default ?